gradle/libs.versions.toml

```diff
@@ -41,6 +41,7 @@ glide = "4.16.0"
 moshi = "1.15.0"
 mockito = "5.8.0"
 forkhandlesBom = "2.12.2.0"
+pluginDependencyGuard = "0.4.3"
```

Comment:
The other Gradle-related entries are at the top. Maybe move it there and remove the "plugin" prefix?

Reply:
The version catalog could be cleaned up, but I would do that as seperate task.